bokeh/client/session.py

```diff
@@ -153,11 +153,16 @@ def __init__(self, session_id=DEFAULT_SESSION_ID, io_loop=None, url=DEFAULT_SERV
         self._connection = ClientConnection(session=self, io_loop=io_loop, url=url)
 
         self._current_patch = None
+        self._callbacks = []
 
     def _attach_document(self, document):
         self._document = document
         self._document.on_change(self._document_changed)
 
+    def add_periodic_callback(self, cb, callback_time):
```

Comment:
@havocp would you just stick the new periodic callback in _callbacks or do we want to differentiate the callbacks that "live in the loop" (now we have periodic but we could even thing a timeout or different ones in future...)? I'd probably go with creating a new _periodic_callbacks or _<whatever>_callbacks to avoid  having to deal with conditions to select how to manage different types of callbacks on a document (i.e. being fired on `on_change`) but you may have a stronger opinion due to some internals that I'm foreseeing..